Makefile

```diff
@@ -77,6 +79,16 @@ docker-build: test ## Build the docker image
 docker-push: ## Push the docker image
 	docker push ${IMG}
 
+helm-docs: ## Generate helm docs
+	cd $(HELM_DIR); \
+	docker run --rm -v $(shell pwd)/$(HELM_DIR):/helm-docs -u $(shell id -u) jnorwood/helm-docs:latest
+
+crds-to-chart: # Copy crds to helm chart directory
+	cp $(CRD_DIR)/*.yaml $(HELM_DIR)/templates/crds/; \
+	for i in $(HELM_DIR)/templates/crds/*.yaml; do \
+		sed -i '1s/.*/{{- if .Values.installCRDs }}/;$$a{{- end }}' $$i; \
```

Comment:
Also, this [doesn't work](https://unix.stackexchange.com/questions/13711/differences-between-sed-on-mac-osx-and-other-standard-sed) on the FreeBSD/OSX default `sed` :( 

We could add comment stating the issue:
```
# If you have any errors running this command, it might be due using a non-GNU `sed` binary
# On OSX, please run: brew install gnu-sed && export PATH="/usr/local/opt/gnu-sed/libexec/gnubin:$PATH"
```

If anyone has any trouble with other OS, we can add more helpful comments.

Reply:
Yeah making portable sed commands is very difficult, essentially have to move out to a script which checks OS and handles all cases. This copies the charts generated by controller-gen into the helm chart and adds an if statement around the crd's to allow controlling whether the chart should install the crds or not. It's mainly to be used by the CI/CD workflow but I can add a comment.

Reply:
Yeah, I know :( 

What about trying something a bit more portable, like this:
```bash
for i in $(HELM_DIR)/templates/crds/*.yaml ; do \
    cp $i $i.bkp && \
    echo "{{- if .Values.installCRDs }}" > $i && \
    cat $i.bkp >> $i && \
    echo "{{- end }}" >> $i && \
    rm $i.bkp 
done
```

It is a bit more verbose, but more explicit and portable.

What do you think?

Reply:
That makes sense, updated Makefile and added comment.

Reply:
Perfect!